packages/vaadin-dialog/src/vaadin-dialog.js

```diff
@@ -124,13 +123,11 @@ class DialogOverlayElement extends mixinBehaviors(IronResizableBehavior, Overlay
 customElements.define(DialogOverlayElement.is, DialogOverlayElement);
 
 /**
- * `<vaadin-dialog>` is a Web Component for creating customized modal dialogs. The content of the
```

Comment:
Please update JSDoc annotation in `vaadin-dialog.d.ts` to keep it in sync with the `vaadin-dialog.js` file.